lib/k8s/client.rb

```diff
@@ -166,8 +166,8 @@ def update_resource(resource)
 
     # @param resource [K8s::Resource]
     # @return [K8s::Resource]
-    def delete_resource(resource)
-      client_for_resource(resource).delete_resource(resource)
+    def delete_resource(resource, **options)
```

Comment:
Just a note that I hate `**something`... could we at least document what options could be?

Reply:
there's two options:
1. Document the options, needs to be synced with `ResourceClient#delete`
   Maybe actually document the options in `ResourceClient#delete` and use only `@see ResourceClient#delete` here
2. Make the options as named args, needs to be synced with `ResourceClient#delete`

Reply:
I would go with option 1 because there are many other methods with `**` .